src/shared/mount-setup.c

```diff
@@ -512,11 +512,21 @@ int mount_cgroup_legacy_controllers(bool loaded_policy) {
         _cleanup_set_free_ Set *controllers = NULL;
         int r;
 
+        /* Before we actually start deleting cgroup v1 code, make it harder to boot in cgroupv1 mode first.
+         * See also #30852. */
+
         if (!cg_is_legacy_wanted())
                 return 0;
-
-        if (!cg_is_legacy_force_enabled())
-                return -ERFKILL;
+        if (cg_is_legacy_force_enabled())
+                log_warning("Legacy support for cgroup v1 enabled via SYSTEMD_CGROUP_ENABLE_LEGACY_FORCE=1.");
```

Comment:
I think this should also include the same "this will stop being supported" wording, to try and communicate that this workaround is not permanent

Reply:
It says "temporarily". I amended NEWS now as requested by Lennart.

Reply:
Oh, it says so in the other branch. But I think here we don't need this. We are very clear that it's temporary in other places, and once the user has made their decision to add the override, we don't need to repeat ourselves.